src/client/components/EditBlogModal.tsx

```diff
@@ -16,7 +16,7 @@ interface EditBlogModalProps {
 }
 
 const EditBlogModal: React.FC<EditBlogModalProps> = ({ isOpen, setOpen, post, cubeID }) => {
-  const [markdown, setMarkdown] = useState<string>(post ? post.body : '');
```

Comment:
With the more correct typing that body can be null, this complained that a non-string could be passed to the state. Updated the form like title below, which handles both the post being undefined and the body having no value